scout/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Docker Scout
+keywords: scout, supply chain, vulnerabilities, packages, cves
+description: >
+  Docker Scout helps you understand your images and their dependencies
+redirect_from:
+  - /atomist/
+  - /atomist/try-atomist/
+  - /atomist/get-started/
+  - /atomist/configure/settings/
+  - /atomist/configure/advisories/
+  - /atomist/integrate/github/
+  - /atomist/integrate/deploys/
+---
+
+> **Note**
+>
+> Docker Scout is an [early access](../release-lifecycle.md#early-access-ea)
+> product, and requires a Pro, Team, or Business subscription.
+
+Docker Scout is a collection of software supply chain features that appear
+throughout Docker user interfaces and the CLI. These features provide detailed
+insights into the composition and security of container images.
+
+Docker Scout analyzes the image contents and generates a detailed report of
+packages and vulnerabilities that it detects. Docker Scout can also help provide
+you with suggestions for how you can remediate issues discovered by the image
+analysis.
+
+The [image details view](./image-details-view.md) in Docker Desktop and the tag
+details pages on Docker Hub are both powered by Docker Scout.
+
+You can view and interact with Docker Scout from your terminal through the
+`docker scout`
+[plugin for Docker CLI](../engine/reference/commandline/scout_cves.md).
+
+There's also a [web UI](https://dso.docker.com/explore){: target="\_blank"
+rel="noopener" } that you can use to explore additional information about
+images, packages, and CVEs.
+
+## Get started with Docker Scout
+
+To start using Docker Scout, enable any of the following features:
+
+- Advanced image analysis in Docker Hub
+- Artifactory integration
+
+Enabling either feature requires a paid Docker subscription. Once enabled,
+Docker Scout is activated for your organization automatically.
+
+### Advanced image analysis
+
+Advanced image analysis is a feature in Docker Hub which, when enabled, triggers
+a Docker Scout analysis every time you push an image. For more information, see
```

Comment:
Can we clarify that the image analysis will be enabled the first time you push an image (ie. unlike scanning, image analysis is always-on - we should emphasise this)